commerce_coordinator/apps/demo_lms/pipeline.py

```diff
@@ -0,0 +1,31 @@
+"""
+Demo LMS app filter pipeline implementations.
+"""
+
+from openedx_filters import PipelineStep
+
+
+# PipelineSteps augment, filter, or transform data passed into them and return a dict that gets merged into the
+# `**kwargs` passed to the next step in the pipeline.  The merge is done using
+# [dict.update()](https://docs.python.org/3/library/stdtypes.html#dict.update)
+#
+# By overriding `run_filter` and adding expected arguments to the signature, we get a small amount checking that the
+# name, for example here "sample_data", was among the arguments passed in.
+class SomeData(PipelineStep):
```

Comment:
Would it be clearer to rename this class to `AddSomeData`? (And correspondingly, rename `SomeMoreData` to `AddSomeMoreData`?)